ci/dump-logs.sh

```diff
@@ -16,8 +16,25 @@
 
 set -eu
 
+readonly BUILD_OUTPUT="build-output/cached-${DISTRO}-${DISTRO_VERSION}"
 # Dump the emulator log file. Tests run in the google/cloud/bigtable/tests directory.
+echo
 echo "================ emulator.log ================"
-cat build-output/cached-${DISTRO}-${DISTRO_VERSION}/google/cloud/bigtable/tests/emulator.log >&2 || true
+cat "${BUILD_OUTPUT}/google/cloud/bigtable/tests/emulator.log" >&2 || true
+echo
 echo "================ instance-admin-emulator.log ================"
-cat build-output/cached-${DISTRO}-${DISTRO_VERSION}/google/cloud/bigtable/tests/instance-admin-emulator.log >&2 || true
+cat "${BUILD_OUTPUT}/google/cloud/bigtable/tests/instance-admin-emulator.log" >&2 || true
+
+readonly ABI_CHECK_REPORTS=$(find ${BUILD_OUTPUT} -name 'compat_report.html')
```

Comment:
Quote RHS on this line and next line, unless this breaks the file iteration below.

Reply:
Done.